src/DynamoCoreWpf/Views/Core/NoteView.xaml

```diff
@@ -132,7 +132,7 @@
             Visibility="{Binding Path=PinnedNode, Converter={StaticResource nullToVisibiltyConverter }, UpdateSourceTrigger=PropertyChanged}"
             >
             <Image
-                Source="/DynamoCoreWpf;component/UI/Images/Pin_NoBackground_16px.png"
+                Source="/DynamoCoreWpf;component/UI/Images/Pin_NoBackground_16_16.png"
```

Comment:
This change is for?

Reply:
the image was renamed on this branch by someone.